extensions/wikia/EmbeddableDiscussions/styles/SettingsModal.scss

```diff
@@ -0,0 +1,106 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+
+.embeddable-discussions-settings-modal.modal {
```

Comment:
For the scss in the entire module, I've generally restricted how many levels to nest (this is also the reason for the long selector names). I generally nest extra for extra attributes such as hover, active etc.

I don't think it makes sense to mess around with the nesting so much just for this one file, as it would make it explicitly different from the other files in the same extension. Of course, now that ownership of the module is being transferred, you are free to refactor, but in either case I think it belongs in a separate PR.